tests/regression/36-apron/22-traces-write-centered-vs-meet-mutex.c

```diff
@@ -22,11 +22,11 @@ void *t_fun(void *arg) {
     y = h;
     __goblint_check(y <= x);
     pthread_mutex_lock(&B);
-    __goblint_check(x == y); // TODO (mutex-meet succeeds, write unknown)
+    __goblint_check(x == y); // mutex-meet succeeds, (now-defunct) write unknown
     pthread_mutex_unlock(&B);
     i = x + 31;
     z = i;
-    __goblint_check(z >= x); // TODO (write succeeds, mutex-meet unknown)
+    __goblint_check(z >= x); // TODO ((now-defunct) write succeeds, mutex-meet unknown)
```

Comment:
Do we want to remove `write` or make an issue for it? I think `write` and `protection` are both a bit dubious.

Reply:
`write` doesn't even exist anymore for relational, hence "now-defunct".

I wouldn't call `protection` "dubious". It's just a very straightforward (and inefficient) attempt, that's not wrong per-se (except for the documented unsoundness w.r.t escapes, which is just an implementation limitation). It could be quite interesting with ELINA, though, if it automatically decomposes the massive relation.